src/apps/intel/intel10g.lua

```diff
@@ -23,7 +23,18 @@ local timer = require("core.timer")
 local bits, bitset = lib.bits, lib.bitset
 local band, bor, lshift = bit.band, bit.bor, bit.lshift
 
-num_descriptors = 1024
+local num_descriptors = 1024
+function ring_buffer_size (arg)
```

Comment:
This looks like its a global setting, shouldn't this be per PCI device / app instance? Maybe even a configuration option of the driver app?